tracer/test/test-applications/integrations/Samples.OpenTelemetrySdk/Program.cs

```diff
@@ -165,6 +165,9 @@ private static void RunSpanUpdateMethods(TelemetrySpan span)
 
         using (var innerSpan = _tracer.StartActiveSpan("InnerSpan"))
         {
+            // if we don't set the span as an error we won't check it for exception event
+            // so if we only do RecordException, we wouldn't copy that info over as the span isn't marked as an Error
+            innerSpan.SetStatus(Status.Error.WithDescription("Something went wrong"));
             innerSpan.RecordException(new ArgumentException("Example argument exception"));
             innerSpan.UpdateName("InnerSpanUpdated");
```

Comment:
This refactoring might make the code look a little more idiomatic, but don't feel required to include this:

```suggestion
            try
            {
                throw new ArgumentException("Example argument exception");
            }
            catch (Exception ex)
            {
                innerSpan.SetStatus(Status.Error.WithDescription(ex.Mesage));
                innerSpan.RecordException(ex);
                innerSpan.UpdateName("InnerSpanUpdated");
            }
```